### DISCUSSION TITLE
New feature ideas

### DISCUSSION THREAD

**Discussion by serialfuzzer** [IamLizu/wpdetect#6] [Ideas]
**Features that I think could be worked upon**

   1. Increasing speed and efficiency
   2. Introducing a history mode where users will be able to access all the websites they've scanned
   3.  Feature to store everything in a log including error messages
   4. --enumerate-plugins feature that can be used to enumerate plugins
   5. REST API: --server flag will start up wpdetect server which could be used to scan URLs over HTTP.
   6. --scan: This flag would scan for basic stuffs such as wp-config backups in websites using wordpress

**Comment by IamLizu** [comment]
So I think #21 contributes to the idea 1.

**Comment by IamLizu** [comment]
3 looks like a good one to pick first because of its simplicity and perhaps we should have this from the start.
